apps/ui/src/components/Ui/Markdown.vue

```diff
@@ -1,21 +1,39 @@
 <script setup lang="ts">
+import hljs from 'highlight.js';
 import { computed } from 'vue';
 import { Remarkable } from 'remarkable';
+import { icons } from '@iconify-json/heroicons-outline';
 import { linkify } from 'remarkable/linkify';
 import { getUrl } from '@/helpers/utils';
 
 const props = defineProps<{
   body: string;
 }>();
 
+const uiStore = useUiStore();
+const { copy } = useClipboard();
+
 const remarkable = new Remarkable({
   html: false,
   breaks: true,
   typographer: false,
-  linkTarget: '_blank'
+  linkTarget: '_blank',
+  highlight: function (str, lang) {
+    if (lang && hljs.getLanguage(lang)) {
+      try {
+        return hljs.highlight(str, { language: lang }).value;
+      } catch (err) {}
```

Comment:
nit. we migrated to using `e` as a name for error in catch block (we have one left it looks like though :D)